contracts/test/token/woeth.js

```diff
@@ -105,7 +106,18 @@ describe("WOETH", function () {
       await expect(woeth).to.have.approxBalanceOf("150", oeth);
       await expect(woeth).to.have.a.totalSupply("50");
     });
-
+    it("should decrease with an OETH negative rebase", async () => {
```

Comment:
We [only rebase upwards](https://github.com/OriginProtocol/origin-dollar/blob/master/contracts/contracts/vault/VaultCore.sol#L389-L391). If the protocol incurs a loss the rebases are "paused" until enough capital is earned for a next positive rebase

Reply:
I was thinking of adding it so that the wrapper would continue to work even if the OETH implementation changed and allowed negative rebasing. But it's true that if the implementation changes, there will probably be stuff to modify in the wrapper anyway, so no need to create tests for a scenario that isn't at the moment ahah.

I'm removing it!